### ISSUE TITLE
InsertAndGernerateKey return value of first column instead of the auto-increment-column

### ISSUE THREAD

**Issue by saeedfielmann** [kotlin-orm/ktorm#210]
By inserting a row into a table with auto-increment-key and using insertAndGenerateKey method, it just returns the value of the first column which must not necessarily be the auto-increment column (id forexample).

I guess it should look for the non compound primary column and return that value instead of value of the first column
or it could be possible the define a custom column for use of auto-generate-key in this method

**Comment by vincentlauvlwj** [comment]
Yes, I agree with you. 

But I don't know how to do that, because the returned column names in the result set is not well-define, and different databases can have different formats, that's why I have to assume the first column is the auto-increment column.

**Comment by saeedfielmann** [comment]
That's right, maybe for the sake of simplicity and flexibility the function InsertAndGenerateKey can accept a user-defined column as parameter, which it can be first, but it can also be changed to another column by column name
